clearly-defined/src/main/kotlin/ClearlyDefinedService.kt

```diff
@@ -409,6 +410,19 @@ interface ClearlyDefinedService {
     @POST("definitions")
     fun getDefinitions(@Body coordinates: Collection<String>): Call<Map<String, Defined>>
 
+    /**
+     * Search for existing definitions based on the [pattern] string provided. The pattern string should contain
+     * the parts of the coordinates (typically namespace, name, and version) relevant for the search. Result is a
+     * list with the ClearlyDefined URIs to all the definitions that are matched by the pattern.
+     *
+     * Note that the _/definitions_ GET endpoint behaves differently depending on the parameters passed in. If only
+     * a pattern string is provided, result is a list of URIs - this use case is covered by this function. It is
```

Comment:
"..., *the* result is ..."

Reply:
Reworded and dropped the part which is not relevant to this function.